GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -4661,6 +4717,40 @@ private async Task<(bool useLongValueColumn, bool valueChanged, bool deleteValue
                     wiserItemDetail.Id = previousField.Id;
                 }
             }
+            
+            if (previousField != null && wiserItemDetail.Id == 0)
+            {
+                wiserItemDetail.Id = previousField.Id;
+            }
+            
+            if (!isNewlyCreatedItem && !previousItemDetails.Any())
+            {
+                DataTable queryResult;
+                
+                if (wiserItemDetail.IsLinkProperty && wiserItemDetail.ItemLinkId > 0)
+                {
+                    queryResult = await databaseConnection.GetAsync($@"SELECT id 
+                                                        FROM {WiserTableNames.WiserItemLinkDetail} 
```

Comment:
WiserItemLinkDetail kan ook een prefix hebben in de GCL, graag dit nog toevoegen. Dit wordt al op plaatsen in de code gedaan waar dit afgekeken kan worden.

Reply:
https://app.asana.com/0/1200346761113317/1204562731570674/f